charts/kube-prometheus-stack/templates/alertmanager/podDisruptionBudget.yaml

```diff
@@ -16,6 +16,5 @@ spec:
   {{- end }}
   selector:
     matchLabels:
-      app: alertmanager
```

Comment:
It should probably be updated to match https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/alertmanager/service.yaml#L47

Reply:
Updated